cypress/integration/spec.js

```diff
@@ -0,0 +1,34 @@
+/// <reference types="Cypress" />
+
+import { SearchPage } from "../page-objects/search-page"
+
+// @ts-check
+const searchPage = new SearchPage()
+describe('Article search', () =>{
+  beforeEach(() =>{
+    searchPage.navigateToSearchPage()
+    searchPage.search("catt")
+  })
+
+  it('search should show results', () => {
+    searchPage.results().first()
+      .should('have.text', "Cattaraugus County, New YorkCounty in New York")
+  })
+
+  it('results with image should show image', () => {
+    cy.get('.result').first()
```

Comment:
All instances of `cy.get('.result')` should also be updated to use `searchPage.results()`

Reply:
was fixed before but forgot to push the commit